src/resizertopbound.js

```diff
@@ -0,0 +1,148 @@
+import {
+	getAbsoluteBoundaryPoint
+} from './utils';
+
+import ObservableMixin from '@ckeditor/ckeditor5-utils/src/observablemixin';
+import mix from '@ckeditor/ckeditor5-utils/src/mix';
+import global from '@ckeditor/ckeditor5-utils/src/dom/global';
+
+/**
+ *
+ * This resizer is top bound, and expands symmetrically towards left, right and increasingly fast toward bottom
+ * (to compensate for top anchoring).
+ *
+ *
+ *
+ */
+
+/**
+ * Implements a resizer that enlarges/shrinks in all directions.
+ *
+ * @class ResizerCentral
+ */
+export default class ResizerCentral {
+	constructor( context, options ) {
+		this.context = context;
+		this.options = options || {};
+	}
+
+	attach() {}
+
+	begin() {
+		const clientRect = this.context._getResizeHost().getBoundingClientRect();
```

Comment:
Please use the `Rect` utility class instead of `getBoundingClientRect` directly in the entire PR.